### ISSUE TITLE
Open a new tab when last tab is closed

### ISSUE THREAD

**Issue by UnsolvedCypher** [gnome-terminator/terminator#56]
I think it would be helpful to have an option that would automatically open a new tab once the last tab in the terminal is closed. I always have a Terminator instance hidden in the background that I bring up when I need, and I sometimes close the last tab by accident and I have to restart Terminator. This feature is present in the Tilda terminal emulator as well as some browsers, so I think it would be great for Terminator to support it as well.

**Comment by lazyfrosch** [comment]
Nice idea, should be a config option, and the user should still be able to quit via menu then

**Comment by mattrose** [comment]
Good idea, and might give me motivation to tackle refactoring the multiple "wrapcloseterm()" methods.
